apple/OmnivoreKit/Sources/App/Views/Home/HomeFeedViewIOS.swift

```diff
@@ -198,7 +226,13 @@ import Views
             }
             if #available(iOS 15.0, *) {
               link
-                .swipeActions(edge: .trailing, allowsFullSwipe: true) {
+                .swipeActions(edge: .trailing, allowsFullSwipe: false) {
```

Comment:
I think we should remove this one. Just the long press should be enough, and with this user's can't do the full edge to edge swipe to archive, which is probably their most common action.